media/css/firefox/built-for-you/landing.scss

```diff
@@ -0,0 +1,206 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@use '~@mozilla-protocol/core/protocol/css/includes/lib' as * with($image-path: '/media/protocol/img');
+
+@import '~@mozilla-protocol/core/protocol/css/components/section-heading';
+@import '~@mozilla-protocol/core/protocol/css/components/video';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-firefox';
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+
+
+.built-for-you {
+    color: $color-black;
+
+    h1, h2, h3 {
+        color: $color-black;
+    }
+}
+
+// Override Protocol wordmark
+.mzp-c-wordmark.mzp-t-wordmark-md.mzp-t-product-firefox {
+    background-image: url('/media/img/logos/firefox/logo-word-hor.svg');
+}
+
+// Override Protocol product button
+.c-button-download-thanks .mzp-c-button.mzp-t-product {
+    background-color: $color-black;
+
+    &:hover,
+    &:focus,
+    &:active {
+        background-color: $color-white;
+        color: $color-black;
+        border-color: $color-black;
+    }
+}
```

Comment:
**issue:** (non-blocking) I know we're about to hit a redesign, but changing the buttons to black for this page does feel kinda inconsistent with our regular pages?

Reply:
There's also a blue CTA in the nav still (not not a blocker):

![image](https://github.com/user-attachments/assets/0e13d406-02f7-4a8e-af67-d1c3e1451bf1)

Reply:
> **issue:** (non-blocking) I know we're about to hit a redesign, but changing the buttons to black for this page does feel kinda inconsistent with our regular pages?

I felt the same, but it's what was designed and approved so clearly intentional 🤷